CMakeLists.txt

```diff
@@ -235,13 +235,21 @@ if(NOT nlohmann_json_FOUND)
       OFF
       CACHE INTERNAL "")
   set(JSON_Install
-      OFF
+      ON
       CACHE INTERNAL "")
```

Comment:
Should we remove the `INTERNAL` attribute on the variable `JSON_Install` as we expect that the user could pass it into command line? Also provide a simple doc string here instead of leaving it as empty?

Reply:
I borrowed this exact syntax based on `nlohmann::json` documentation. Described here:
https://json.nlohmann.me/integration/cmake/#supporting-both

Internal does not matter much - you can still provide an option to toggle it `OFF` in command line. It works well. And I think the only case when you'd want that OFF is MAYBE for Linux and Mac? And only if you build without zPages. This is minor detail, so I'd rather not bother changing it.